src/main/java/gregtech/loaders/recipe/chemistry/DistillationRecipes.java

```diff
@@ -94,6 +95,7 @@ public static void init() {
         DISTILLATION_RECIPES.recipeBuilder()
                 .fluidInputs(SeedOil.getFluid(24))
                 .fluidOutputs(Lubricant.getFluid(12))
+                .cleanroom(CleanroomType.CLEANROOM)
```

Comment:
why Seed Oil distillation requires cleanroom...?
If it is intended, why Fish Oil distillation does not require cleanroom?